### ISSUE TITLE
Add a notice (or faq) about paypal clearing

### ISSUE THREAD

**Issue by lefnire** [HabitRPG/habitica#2384]
Message from a user:

> I just went to look up the paypal transaction and discovered that for some reason they're saying it won't clear for four more days.  If I'd realized that I'd have used a credit card.
> 
> So apparently there's no problem on your end, and I should have known better than to use PayPal for a simple $5 transaction.  If this is a common thing, though, it would probably be a good thing to put into the FAQ or on the donate page.  I spent a while trying to figure out what was wrong before I figured out who to email (and found the real problem).

This might explain why we get so many paypal problem emails

**Comment by Jaegermeiste** [comment]
It took about 20 minutes for my recent transaction to clear (which I just found out in the course of this writing). It's kinda messed up that I had to find the reason on GitHub... There should be a yellow banner warning you about the possible delays in processing if you click PayPal instead of credit. While I (obviously) intend to support the developers, it is incredibly upsetting that my first feelings after the transaction are confusion, annoyance, and regret (and hitting F5 3 or 4 times). I even briefly considered cancelling the transaction.

A banner in the donation popup would be a simple band-aid until whatever (if any) processing issues exist with PayPal are ironed out. It may be a minor issue in terms of the tech, but may be a bigger psychological issue than you realize.
